scaletest/dashboard/rolltable.go

```diff
@@ -0,0 +1,304 @@
+package dashboard
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/codersdk"
+)
+
+// allActions is a table of actions to perform.
+// D&D nerds will feel right at home here :-)
+// Note that the order of the table is important!
+// Entries must be in ascending order.
+var allActions rollTable = []rollTableEntry{
```

Comment:
I'm not sure that I buy the idea of rolling a dice. Isn't it better to perform a set of separate tests for each of these cases?

Reply:
Do you mean like, save all the calls when hitting the "Users" page and gather them into 1 "scenario" that gets run?

Reply:
Correct - we could build out each action to include multiple fetches in parallel.
For now I'm focusing on keeping it simple so that we don't have to constantly play catchup with the frontend.

Reply:
The "table" construct is primarily an implementation detail to make it easier to weight the probability of each scenario. I could imagine fetching HTTP logs of an existing instance, and setting the priority of each endpoint accordingly.

Reply:
Ok